### ISSUE TITLE
haptic feedback part vibrates when playing videos

### ISSUE THREAD

**Issue by tashijayla** [deckerst/aves#793]
**Describe the bug**

When playing video the haptic feedback part vibrates to every sound the video makes

**To Reproduce**
1. Install and open the app
2. Play any video

**Expected behavior**
Video should play without activating the vibration' haptic' part of the phone 

**System information and logs:**

Package: deckers.thibault.aves
Installer: com.looker.droidify
Aves version: 1.9.6-izzy, build 10701
Flutter: stable 3.13.4
Android version: 14, API 34
Android build: UP1A.231005.007.2023101300
Device: Google Pixel 5
Geocoder: not available
Mobile services: ready
System locales: en_GB
Storage volumes: /storage/emulated/10/
Storage grants: 
Error reporting: false
OS GrapheneOS 

Thank you for making the best gallery app ever 🤩

**Comment by deckerst** [comment]
Are you sure this is what's happening? It sounds like such a strange bug... Also, there are so many other users, even using GrapheneOS, that I think it's not something other people experience.

For example, if you mute the video, do you still get the haptic feedback? If you play a music video, do you get constant hapyic feedback? Are you sure it's not just your phone in mechanical resonance with the sound produced?

**Comment by tashijayla** [comment]
> Are you sure this is what's happening? It sounds like such a strange bug... Also, there are so many other users, even using GrapheneOS, that I think it's not something other people experience.
> 
> For example, if you mute the video, do you still get the haptic feedback? If you play a music video, do you get constant haptic feedback? Are you sure it's not just your phone in mechanical resonance with the sound produced?

If I turn the volume down their is less Vibrations, if I turn the volume up their is more Vibrations. I've tested playing the same video's on YouTube and in VLC and more and my phone definitely vibrates much more when playing videos in Aves, video's also play with more bass in Aves. Even when I put the Max bass settings on other apps the phone doesn't vibrate as much. I wish it was a bit easier to show.

I completely understand if you would like to wait until more people report this as a bug until you have a deeper look.

Thanks for your quick response 😊

**Comment by tashijayla** [comment]
I should have told you that it didn't start happening with the last version it started happening 2 or 3 app versions ago. If it would help I could try out older versions until I find the last version that didn't activate the vibration haptic feedback engine

**Comment by deckerst** [comment]
If it started appearing a few versions ago, it's likely related to the change of video engine in v1.9.0.
Could you try the following?
1) go to https://github.com/media-kit/media-kit/releases/tag/media_kit-v1.1.8
2) in Assets, find the `media_kit_test_android-....apk` matching your device architecture (probably `media_kit_test_android-arm64-v8a.apk`)
3) install and open the test app
4) play one of your own videos (as I remember there's a button in the bottom right corner for this)

Do you get the same issue?

**Comment by tashijayla** [comment]
Yes I experience the same issue. Would you like me to create a bug report and post it on that apps list of issues?

**Comment by deckerst** [comment]
Yes, you could file a report on the `media_kit` repo. That package is based on the [MPV player](https://play.google.com/store/apps/details?id=is.xyz.mpv), so you could try your video with it first, and tell them whether you also have the issue with it, when you file the report.

**Comment by deckerst** [comment]
Did you report it to them?

**Comment by tashijayla** [comment]
@deckerst I'm happy to say 

mpv version 2023-11-30.
Is.xyz.mpv version code 8033.
TargetSdk 33 
MinSdk 21

Has completely fixed my issue, so I'm positive that once you update the video Aves SDK videos should sound great again  😀🥳

**Comment by deckerst** [comment]
Nice! It will probably take some time for it to trickle down to Aves (mpv > media_kit > aves) but it's good to know it's fixed upstream.

I will leave this issue open until then.

**Comment by tashijayla** [comment]
It would seem the issue was only on the Pixel 5 because now I've updated to the Pixel 7, it's working as intended. May I close this issue as completed?

**Comment by deckerst** [comment]
Glad to know it works for you now. I'll close the issue.
